TriblerGUI/widgets/discoveredpage.py

```diff
@@ -36,4 +36,4 @@ def initialize_discovered_page(self, gui_settings):
 
     def load_discovered_channels(self):
         self.controller.model.reset()
-        self.controller.load_channels(1, 50)  # Load the first 50 discovered channels
+        self.controller.perform_query(start=1, end=50)  # Load the first 50 discovered channels
```

Comment:
Probably not related to changes in this PR, but we are using start and end here as variable names and in `channelpage.py` we are using first and last. We should make these names consistent.

Reply:
Agree.